products/epinio/list/namespaces.vue

```diff
@@ -128,7 +128,7 @@ export default {
         <div slot="body">
           <LabeledInput
             v-model="value.name"
-            :min-height="'90px'"
+            :min-input-height="'90px'"
```

Comment:
This would then revert as well

Reply:
If it's min-height it conflicts with a property in another component